wordpress-mobile/WordPress-iOS
# Configure Info.plist and entitlements for notification service extension (Issue/9888)

Fixes #9888. This needs to precede the extension implementation, as API client access is required.

To test:

- Inspect the individual files changed. Juxtapose the proposed changes with counterparts on existing extensions. Verify that the correct App Identifier prefix is used by each variant.
- Verify that the app continues to build & run.
- Verify that existing tests pass.
